jest-unit-config.js

```diff
@@ -0,0 +1,6 @@
+export const displayName = 'Unit Tests';
+export const testEnvironment = 'jsdom';
+export const testMatch = ['**/src/tests/*.spec.ts'];
```

Comment:
Os testes estarão no próprio diretório do módulo/componente/página, não?
Se for o caso, esse caminho está incorreto.